config/ocp_aws/install/fence_aws_credentials_request.yaml

```diff
@@ -0,0 +1,45 @@
+# similar to https://github.com/openshift/machine-api-operator/blob/master/install/0000_30_machine-api-operator_00_credentials-request.yaml#L1-L6
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
```

Comment:
I'm afraid this won't work, it won't be installed. OLM supports a limited set of resource types only. See switch cases at [1] and few CRD types at [2].

[1] : https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/operators/catalog/operator.go#L2070
[2]: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/operators/catalog/supportedresources.go#L15

Reply:
Do you mean that the [`openshift-aws-cloud-fencing`](https://github.com/medik8s/fence-agents-remediation/pull/20/files/3cc361ad0c09cc88329d5a22856a96f0f2327d03#diff-6c27ee83cb33364a46c46291cde6b29cd14061ee251d05d7c7d8884d8390e9f6R5) CR won't be created on OpenShift CI?
Looking on the logs from E2E it looks like it was indeed created ([lines 103-107](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/medik8s_fence-agents-remediation/20/pull-ci-medik8s-fence-agents-remediation-main-4.12-openshift-e2e/1666095122816700416#1:build-log.txt%3A107)), but I would wait for a [fixed `fence_aws`](https://bugzilla.redhat.com/show_bug.cgi?id=2183158) FA to verify my understanding in practice :D 

>Downloading  mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.0/openshift-client-linux-4.13.0.tar.gz
mkdir -p /go/src/github.com/medik8s/fence-agents-remediation/bin/oc/4.13.0/oc-extracted 2>/dev/null || true
tar -xzf /go/src/github.com/medik8s/fence-agents-remediation/bin/oc/4.13.0/oc-tar -C /go/src/github.com/medik8s/fence-agents-remediation/bin/oc/4.13.0/oc-extracted
/go/src/github.com/medik8s/fence-agents-remediation/bin/kustomize/v4@v4.5.7/kustomize build config/ocp_aws | /go/src/github.com/medik8s/fence-agents-remediation/bin/oc/4.13.0/oc-extracted/oc apply -f -
credentialsrequest.cloudcredential.openshift.io/openshift-aws-cloud-fencing created

Reply:
oh ok, you create the CredentialsRequest yourself. I thought you want to put it into the bundle and let OLM install it. All good then 🙂